versions/v1/units.json

```
@@ -8110,6 +8314,27 @@
     "source": "qudt.org",
     "sourceReference": "https://qudt.org/vocab/unit/MegaL"
   },
+  {
+    "externalId": "volume:mega-m3",
+    "name": "Mega-M3",
+    "quantity": "Volume",
+    "longName": "Million Standard Cubic Meter",
+    "aliasNames": [
+      "mega cubic meter",
+      "mega cubic metre",
+      "mmcm",
+      "MMCM",
+      "mmscm",
+      "MMscm"
+    ],
+    "symbol": "MMscm",
+    "conversion": {
+      "multiplier": 1.0e+6,
+      "offset": 0.0
+    },
+    "source": "Energistics: Unit of Measure Standard - https://www.energistics.org/energistics-unit-of-measure-standard",
```

```suggestion
    "source": "Energistics: Unit of Measure Standard (revision June 2014) - https://www.energistics.org/energistics-unit-of-measure-standard",
```
Here as well, let's mention which revision we used as a source